modules/admin/procedure-understanding-no-proxy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="understanding-no-proxy"]
+= Understanding the `NO_PROXY` exclusion rules
+
+`NO_PROXY` is a comma or space-separated list of hostnames or IP addresses, optionally with port numbers. If the input URL matches any of the entries listed in `NO_PROXY`, then that URL will be fetched by a direct request (i.e., bypassing the proxy settings).
+
+Note that the default value for `NO_PROXY` in {product-very-short} is `localhost,127.0.0.1`. If you want to override it, please make sure to also include at least `localhost` or `localhost:7007` in the list; otherwise, the {product-very-short} Backend may not work correctly.
```

Comment:
```suggestion
[NOTE]
	====
	The default value for `NO_PROXY` in {product-very-short} is `localhost,127.0.0.1`. If you want to override it, include at least `localhost` or `localhost:7007` in the list. Otherwise, the {product-very-short} Backend may not work correctly.
	====
```

Thanks for the suggestion. I've updated to remove the links to `http://localhost` and `http://localhost:7007`, as we don't need to have links displayed here.